src/groups/mqb/mqbc/mqbc_clusterutil.cpp

```diff
@@ -1390,7 +1390,7 @@ void ClusterUtil::updateAppIds(ClusterData*                    clusterData,
                        << queueAdvisory << ", rc: " << rc;
     }
     else {
-        BALL_LOG_INFO << "Unregister appIds " << printRemoved
+        BALL_LOG_INFO << "Unregistered appIds " << printRemoved
                       << " and registered appIds " << printAdded << " for '"
```

Comment:
`ledger->apply()` is an async update, so unsure what is best to print here. Also, the `uri` logged at the end of this log might be empty, which is not intuitive. Maybe that should print domain or uri, whichever was used?